src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java

```diff
@@ -134,7 +139,12 @@ public String getResolvConfPath() {
         return volumes == null ? null : volumes.getBinds();
     }
 
+    /**
+     * @deprecated As of {@link RemoteApiVersion#VERSION_1_20}
+     * use {@link #getMounts()} instead
+     */
     @JsonIgnore
+    @Deprecated
```

Comment:
Will it be not available in JSON response or it just deprecated by docker and fill be removed in future?